angular-site/src/app/layout/grid-list/grid-list-dynamic-example.html

```diff
@@ -1,6 +1,7 @@
 <h2 class="example-h2">Dynamic grid-list</h2>
 <mat-grid-list id="dynamic-grid-list" cols="4" rowHeight="100px">
   <mat-grid-tile
+      id="{{ 'dynamic-grid-list-' + tile.text }}"
```

Comment:
On the page dynamic element looks like "dynamic-grid-list-One" could you please change the id with low case letters only.